### PR TITLE
Add new flags and functionality to ls command

### PR DESCRIPTION
**Pull request by n0npax**
fix some rustfmt issues in file ls.rst
fix fail when symlink is broken (will print info now)
add showing symbolic link destination (-l)
add possibilities to show date of:
* modification [--modified-date] (-l)
* access [--accessed-date]
* creation [--created-date]

### CODE REVIEW

File: src/bin/ls.rs [hunk 1]
```diff
@@ -39,16 +39,23 @@ OPTIONS
         reverse order while sorting
     -R, --recursive
         list subdirectories recursively
+    --modified-date
```
Comment (ca1ek) [inline src/bin/ls.rs]: Short forms for these commands would be great, `--mdate`, `--adate`, `--cdate`

**Comment by jackpot51** [comment]
I can't merge this, due to `chrono` not being supported in Redox (I think)

Can you format the date using what the `date` program does?

File: src/bin/ls.rs (hunk 1 above)
Comment (n0npax) [inline src/bin/ls.rs]: added :)

**Comment by n0npax** [comment]
I'll try to replace chrono with stuff from _date_ tomorrow.

**Comment by jackpot51** [comment]
Cool, thanks!

**Comment by jackpot51** [comment]
Thanks - looks good!

**Comment by n0npax** [comment]
Thanks :)

**Pull request merged**